Mlem/App Constants.swift

```diff
@@ -86,4 +90,7 @@ struct AppConstants {
     static let fullMarkReadSymbolName: String = "envelope.open.fill"
     static let emptyMarkUnreadSymbolName: String = "envelope.open"
     static let fullMarkUnreadSymbolName: String = "envelope.fill"
+    
+    // MARK: - Other
+    static let pictureEmoji: [String] = ["🎆", "🎇", "🌠", "🌅", "🌆", "🌁", "🌃", "🌄", "🌉", "🌌", "🌇", "🖼️", "🎑", "🏞️", "🗾", "🏙️"]
```

Comment:
It worries me a little that there will be posts with these emojis in them and people might get confused about why no images appear.  But that's probably true about any character we swap out for.